internal/ascprails/postgres.go

```
@@ -0,0 +1,532 @@
+
+func (s *PostgresStore) enqueueOnce(ctx context.Context, input EnqueueInput) (Job, bool, error) {
+	now := s.clock().UTC()
+	fingerprint, headersJSON, offerJSON, paymentJSON, bindingJSON, err := encodeInput(input)
+	if err != nil {
+		return Job{}, false, err
+	}
+	tx, err := s.db.BeginTx(ctx, &sql.TxOptions{Isolation: sql.LevelSerializable})
+	if err != nil {
+		return Job{}, false, err
+	}
+	defer tx.Rollback()
```

golangci-lint (2.12.2)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)